common/Cargo.toml

```diff
@@ -27,7 +27,6 @@ http.workspace = true
 ipnetwork.workspace = true
 macaddr.workspace = true
 mg-admin-client.workspace = true
-dpd-client.workspace = true
```

Comment:
This causes downstream breakage, as compilation of this crate requires omicron-specific build machinery.

Reply:
ah yeah... I feel like CI should catch this, maybe by building a stub project with omicron-common as its only dependency. Could you file an issue for this?

Reply:
https://github.com/oxidecomputer/omicron/issues/5748